src/dmd/parse.d

```diff
@@ -6076,10 +6076,26 @@ final class Parser(AST) : Lexer
             }
         case TOK.import_:
             {
-                AST.Dsymbols* imports = parseImport();
-                s = new AST.ImportStatement(loc, imports);
-                if (flags & ParseStatementFlags.scope_)
-                    s = new AST.ScopeStatement(loc, s, token.loc);
+                /* https://issues.dlang.org/show_bug.cgi?id=16088
+                 * At this point it can either be an import expression
+                 * or an import declaration. See if the next token after
```

Comment:
Add `https://dlang.org/spec/grammar.html#ImportDeclaration` instead of `import declaration`, and ditto for `https://dlang.org/spec/grammar.html#ImportExpression`

Reply:
Having links to the corresponding Spec place that the code is implementing is very handy for reading the code and ensuring it is in sync with the Spec.